runner.py

```diff
@@ -1326,33 +1361,38 @@ def save_stdout_logs(self):
                 """, params=(logs_as_str, self._run_id))
 
 
-    def cleanup(self):
+    def cleanup(self, inline=False):
```

Comment:
I would rename `inline` to `keep_state` or something like that as inline is not really clear what it does

Reply:
renamed to continue_measurement